docs/components/operate/userguide/process-instance-modification.md

```diff
@@ -141,3 +141,7 @@ Some elements do not support specific modifications:
 - **Add token**/**Move tokens to** modifications are currently not possible for elements with multiple running scopes.
 
 ![not-supported-flow-nodes](../../../images/operate/modifications/not-supported-flow-nodes.png)
+
+## Monitor the modification status
```

Comment:
Do you mean "monitor the operation status" here?

Reply:
Modification is a specific type of operation. I think it makes sense to call it "modification operation status". I rephrased it.